src/target.d

```diff
@@ -33,32 +33,26 @@ struct Target
     extern (C++) static __gshared int c_long_doublesize;    // size of a C 'long double'
     extern (C++) static __gshared int classinfosize;        // size of 'ClassInfo'
 
-    template FPTypeProperties(T)
+    extern (C++) template FPTypeProperties(T)
     {
-        enum : real_t
-        {
-            max = T.max,
-            min_normal = T.min_normal,
-            nan = T.nan,
-            snan = T.init,
-            infinity = T.infinity,
-            epsilon = T.epsilon
-        }
+        static __gshared real_t max = T.max;
+        static __gshared real_t min_normal = T.min_normal;
+        static __gshared real_t nan = T.nan;
+        static __gshared real_t snan = T.init;
+        static __gshared real_t infinity = T.infinity;
+        static __gshared real_t epsilon = T.epsilon;
```

Comment:
@MartinNowak - yes, I noticed.  And was thinking of making a note in a talk about it. ;-)